libcontainer/rootfs_linux.go

```diff
@@ -299,6 +299,22 @@ func checkMountDestination(rootfs, dest string) error {
 	invalidDestinations := []string{
 		"/proc",
 	}
+	// White list, it should be sub directories of invalid destionations
```

Comment:
@hqhq 
small typo in the comment, "invalid destionations" -> invalid destinations